proto/chart/v1/send.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+option go_package = "chart/send/v1;stream_charts";
+
+package chart.v1;
+
+import "google/protobuf/timestamp.proto";
+
+message StreamChartsAndDimensions {
+
+  // claim_id, node_id pair is used to identify the Node Instance
+  string claim_id = 1;
+  string node_id = 2;
+  // used to define the position of the Cloud
+  int64 seq_id = 3;
+  int64 prev_seq_id = 4;
+  int64 batch_id = 5;
+  google.protobuf.Timestamp updated_at = 6;
+
+  // vmq_instance_id broker shard to use for reaching the agent
+  // cloud injects this information.
+  int32 vmq_instance_id = 7;
+  // mqtt_broker_addr shard to use for reaching the agent
+  // cloud injects this information.
+  string mqtt_broker_addr = 8;
```

Comment:
not sure if these fields are necessary since this message is going to be delivered to and processed by the agent.
```suggestion
```

Reply:
I kinda get it but not completely @lvrach. 
The command will be placed inside the MQTTEnvelope as payload. In the same level with payload there will also be the `claim_id` field.

- I understand that the `vmq_instance_id` will actually be part of the the Pulsar topic name, so this is ok.
- What about the broker address, will the Output service know this and forward the message to the correct broker address?

Reply:
I removed the fields from the proto definition.